app.py

```diff
@@ -0,0 +1,124 @@
+import connexion
+from config import Config
+from connexion.resolver import MethodViewResolver
+from flask import Flask
+from flask import request
+from flask_assets import Environment
+from flask_talisman import Talisman
+from frontend.assets import compile_static_assets
+from fsd_utils.logging import logging
+from jinja2 import ChoiceLoader
+from jinja2 import PackageLoader
+from jinja2 import PrefixLoader
+from openapi.utils import get_bundled_specs
+
+
+def create_app() -> Flask:
+
+    # Configure Connexion and Swagger
+    options = {
+        "swagger_url": "/docs",
+        "swagger_ui_template_arguments": {},
+    }
+
+    connexion_app = connexion.FlaskApp(
+        __name__,
+        specification_dir="/openapi/",
+        options=options,
+    )
+
+    connexion_app.add_api(
+        get_bundled_specs("/openapi/api.yml"),
+        validate_responses=True,
+        resolver=MethodViewResolver("api"),
+    )
+
+    # Configure flask_app
+    flask_app = connexion_app.app
+    flask_app.config.from_object("config.Config")
+
+    # Configure jinja templates and static files
+    flask_app.static_folder = "frontend/static/dist/"
+    flask_app.jinja_loader = ChoiceLoader(
+        [
+            PackageLoader("frontend"),
+            PrefixLoader(
+                {"govuk_frontend_jinja": PackageLoader("govuk_frontend_jinja")}
+            ),
+        ]
+    )
+
+    flask_app.jinja_env.trim_blocks = True
+    flask_app.jinja_env.lstrip_blocks = True
+
+    # Initialise logging
+    logging.init_app(flask_app)
+
+    # TODO : Uncomment and setup redis if using redis sessions
+    # Initialise redis sessions
+    # session = Session()
+    # session.init_app(flask_app)
+
```

Comment:
🤔 was this left behind by accident? 😆

Reply:
I've added TODO comments generally on the thinking that this is a template rather than an active repo, so there's bits of optional code that a future dev can uncomment and implement based on whatever their real repo needs. Also for sessions to work this repo would need a redis instance attached to it which it isn't yet